bears/generate_package.py

```diff
@@ -94,24 +103,64 @@ def create_upload_parser():
                         action='store_true')
     parser.add_argument('-u', '--upload', help='Upload the packages on PyPi',
                         action='store_true')
+    parser.add_argument('-c', '--conda', help='Create a conda package',
```

Comment:
Hmm did not notice too many conventions around now. Okay then let's stick to this. @sils1297 Do you think we need to make a codestyle issue for this?